cpp/unreal_plugins/SpEngine/Source/SpEngine/LegacyService.h

```diff
@@ -39,6 +39,24 @@ class LegacyService {
         post_world_initialization_handle_ = FWorldDelegates::OnPostWorldInitialization.AddRaw(this, &LegacyService::postWorldInitializationHandler);
         world_cleanup_handle_ = FWorldDelegates::OnWorldCleanup.AddRaw(this, &LegacyService::worldCleanupHandler);
 
+        unreal_entry_point_binder->bindFuncNoUnreal("legacy_service", "get_random_points",
```

Comment:
This is not safe. `NoUnreal` functions can be called as soon as the RPC server is initialized, but `nav_mesh_` is only initialized in `LegacyService::worldBeginPlayHandler()`. (Also, the new convention I'd like to adhere to is calling these functions `Handlers` rather than `EventHandlers` for brevity.

Reply:
Nice catch! Updated the code to make navmesh related functions bind using `BindFuncUnreal` and associated usage on the python side.